packages/x-charts/src/context/CartesianProvider/computeValue.ts

```diff
@@ -80,16 +78,15 @@ export function computeValue({
   formattedSeries,
   axis: inAxis,
```

Comment:
This `inAxis` is here to avoid naming conflict. Not that the normalization is removed it does not make any  sense.

You can either do this modification, or the opposit (use `inAxis` inside the function instead of `allAxis`)

```suggestion
  axis: allAxis,
```